sdk/schemaregistry/schema-registry-avro/test/utils/mockedRegistryClient.ts

```diff
@@ -61,11 +61,19 @@ export function createTestRegistry(neverLive = false): SchemaRegistry {
   async function getSchemaProperties(
     schema: SchemaDescription,
     _options?: GetSchemaPropertiesOptions
-  ): Promise<SchemaProperties | undefined> {
-    return mapByContent.get(schema.content);
+  ): Promise<SchemaProperties> {
+    const result = mapByContent.get(schema.content);
+    if (result !== undefined) {
```

Comment:
We can use simple truthy for this, can't we? 

```suggestion
    if (result) {
```